static.json

```diff
@@ -0,0 +1,7 @@
+{
+  "root": "dist",
+  "clean_urls": true,
+  "routes": {
+    "/**": "index.html"
+  }
+}
```

Comment:
@kdaglas , you forgot to add a new line here.

Reply:
let me add it